### PR TITLE
Mealie Version 1

### PR DESCRIPTION
**Pull request by cellardoor452**
This is Mealie Version 1 Omni Channel, not compatible with previous versions. Version on CA not updateable.

### CODE REVIEW

**Comment by Roxedus** [comment]
I think this has the wrong file :)

**Comment by Roxedus** [comment]
there is only yt-sub here https://github.com/selfhosters/unRAID-CA-templates/pull/394/files

**Comment by Roxedus** [comment]
Yes, but also ytdl-sub

**Comment by cellardoor452** [comment]
Ok, I think I fixed it, Not quite sure how I did that, sorry about that.

**Comment by cellardoor452** [comment]
Anything else needed? it should be good? And how long after approving do we see in CA? for example don't see the YTDL-SUB yet.

**Comment by catduckgnaf** [comment]
@Roxedus  can you confirm only Mealie is there now?

File: templates/mealiev1.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  <Config Name="Container Variable 10" Target="PYTHONDONTWRITEBYTECODE" Default="1" Mode="" Description="" Type="Variable" Display="advanced-hide" Required="false" Mask="false">1</Config>
+  <Config Name="PUID" Target="99" Default="" Mode="" Description="" Type="Variable" Display="advanced-hide" Required="false" Mask="false">99</Config>
+  <Config Name="PGUID" Target="PGUID" Default="" Mode="" Description="" Type="Variable" Display="advanced-hide" Required="true" Mask="false">100</Config>
+  <Config Name=" WEB_CONCURRENCY" Target=" WEB_CONCURRENCY" Default="" Mode="" Description="" Type="Variable" Display="advanced-hide" Required="false" Mask="false">1</Config>
+  <Config Name="MAX_WORKERS" Target="MAX_WORKERS" Default="" Mode="" Description="" Type="Variable" Display="advanced-hide" Required="true" Mask="false">1</Config>
+  <Config Name="TZ=America/New_York" Target="TZ" Default="" Mode="" Description="" Type="Label" Display="advanced-hide" Required="true" Mask="false">America/New_York</Config>
```
[the remark this answers is not in the document]
Comment (cellardoor452) [reply templates/mealiev1.xml]: Sorry, I think I fixed it and submitted an update

**Review by catduckgnaf — APPROVED** [review]
Changed Made

**Pull request merged**